src/PolykeyAgent.ts

```diff
@@ -67,15 +67,6 @@ type PolykeyAgentOptions = {
   nodes: NodesOptions;
```

Comment:
I don't know if `NodesOptions` makes sense here. The main reason is because options at `PolykeyAgent` isn't a composition of options of subdomains. It's its own thing. I don't want to tie the API of `PolykeyAgent` to the underlying domains. So I'm removing this and replacing it with embedded options. The `NCM` do its own thing though,

Reply:
Removed.